include/ignition/sensors/SegmentationCameraSensor.hh

```diff
@@ -25,13 +25,15 @@
 #include <ignition/common/PluginMacros.hh>
 #include <ignition/common/SuppressWarning.hh>
 #include <ignition/common/Time.hh>
+#include <ignition/transport/Node.hh>
+#include <ignition/transport/Publisher.hh>
+#include <ignition/msgs.hh>
+#include <sdf/sdf.hh>
 #include "ignition/sensors/CameraSensor.hh"
```

Comment:
```suggestion
#include <sdf/sdf.hh>

#include "ignition/sensors/CameraSensor.hh"
```

nit: add whitespace to separate other project headers from local project headers